Tests/JWTKitTests/JWTMigrationTests.swift

```diff
@@ -0,0 +1,53 @@
+import XCTest
+import Crypto
+#if os(Linux)
+import FoundationNetworking
+#endif
+@testable import JWTKit
+
+class JWTKitMigrationTests: XCTestCase {
+    func testVerifyingCryptoKey() async throws {
```

Comment:
An end to end test to validate everything is hooked up properly. Also tested the migrations in the app-store-server-library repo along with my own codebase.